Tasks/AzurePowerShellV4/task.json

```diff
@@ -139,7 +139,11 @@
             "required": true,
             "visibleRule": "TargetAzurePs = OtherVersion",
             "groupName": "AzurePowerShellVersionOptions",
-            "helpMarkDown": "Preferred Azure PowerShell Version needs to be a proper semantic version eg. 1.2.3. Regex like 2.\\*,2.3.\\* is not supported. The Hosted VS2017 Pool currently supports Az module version: 1.0.0"
+            "helpMarkDown": "Preferred Azure PowerShell Version needs to be a proper semantic version eg. 1.2.3. Regex like 2.\\*,2.3.\\* is not supported. The Hosted VS2017 Pool currently supports Az module version: 1.0.0",
+            "validation": {
+                "expression": "isMatch(value, '^[0-9]+\\.[0-9]+\\.[0-9]+$','Multiline')",
```

Comment:
why do we need the \\ delimiter in the expression?

Reply:
With \. , it shows error. That's why I used \\. , it is working fine.